int16   PID
int64   RAM_MB
int16   CPU_Perc
